charts/camunda-platform-alpha/test/unit/console/configmap_test.go

```diff
@@ -50,29 +47,21 @@ func TestConfigMapTemplate(t *testing.T) {
 	})
 }
 
-func (s *configMapTemplateTest) TestContainerShouldSetCorrectIdentityType() {
-	// given
-	options := &helm.Options{
-		SetValues: map[string]string{
-			"console.enabled":                       "true",
-			"global.identity.auth.type":             "MICROSOFT",
-			"global.identity.auth.issuer":           "https://example.com",
-			"global.identity.auth.issuerBackendUrl": "https://example.com",
-		},
-		KubectlOptions: k8s.NewKubectlOptions("", "", s.namespace),
-	}
-
-	// when
-	output := helm.RenderTemplate(s.T(), options, s.chartPath, s.release, s.templates)
-	var configmap corev1.ConfigMap
-	var configmapApplication ConsoleYAML
-	helm.UnmarshalK8SYaml(s.T(), output, &configmap)
-
-	err := yaml.Unmarshal([]byte(configmap.Data["application.yaml"]), &configmapApplication)
-	if err != nil {
-		s.Fail("Failed to unmarshal yaml. error=", err)
-	}
-
-	// then
-	s.Require().Equal("MICROSOFT", configmapApplication.Camunda.Console.OAuth.Type)
+func (s *configMapTemplateTest) TestDifferentValuesInputs() {
+	testCases := []testhelpers.TestCase{
+        {
+            Name: "Container should set correct identity type",
+            Values: map[string]string{
+                "console.enabled":                       "true",
+                "global.identity.auth.type":             "MICROSOFT",
+                "global.identity.auth.issuer":           "https://example.com",
+                "global.identity.auth.issuerBackendUrl": "https://example.com",
+            },
+            Expected: map[string]string{
+                "configmapApplication.camunda.console.oAuth.type": "MICROSOFT",
```

Comment:
I would like to know where this `configmapApplication` prefix is coming from. Is it automatically prepended by the renderTemplate function through terratest?

Reply:
Thats a great comment: With this key the verifyConfigMap function decides if it needs to verify application.yaml formats or envVars.

```
if strings.HasPrefix(keyPath, "configmapApplication.") {
   [...]
   actualValue = getConfigMapFieldValue(configmapApplication, strings.Split(keyPath, ".")[1:])
} else {
   actualValue = strings.TrimSpace(configmap.Data[keyPath])
}
```

Any idea to make this smarter are appreciated!

Reply:
--> Its nothing related to terratest but "self-made"